src/main/java/com/spotlightspace/core/point/dto/GetPointResponseDto.java

```diff
@@ -0,0 +1,22 @@
+package com.spotlightspace.core.point.dto;
+
+import com.spotlightspace.core.point.domain.Point;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class GetPointResponseDto {
+
+    private Long id;
+    private int amount;
+    private String nickname;
+    private GetPointResponseDto(Point point) {
+        this.id = point.getId();
+        this.amount = point.getAmount();
+        this.nickname = point.getUser().getNickname();
+    }
+    public static GetPointResponseDto from(Point point) {
+        return new GetPointResponseDto(point);
+    }
```

Comment:
```suggestion
    private String nickname;
    
    private GetPointResponseDto(Point point) {
        this.id = point.getId();
        this.amount = point.getAmount();
        this.nickname = point.getUser().getNickname();
    }
    
    public static GetPointResponseDto from(Point point) {
        return new GetPointResponseDto(point);
    }
```
필드와 생성자 사이에 개행을 하는 컨벤션이 있습니다.
확인 부탁드립니다.
코드 작성 후 ctrl + alt + l 을 사용하면 편리합니다.